queries.sql

```diff
@@ -21,3 +21,100 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including animals with weights equal to 10.4kg or 17.3kg):
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+--Inside a transaction update the animals table by setting the species column to unspecified.
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+-- Check if the column named species is updated.
+SELECT * from animals;
+
+--Undo all the changes you have made before.
+
+ROLLBACK;
+
+--Check if the table become as it was before the changes were made.
+
+SELECT * FROM animals;
+
+--Update the animals table by setting the species column to digimon for all animals that have a name ending in mon
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+--Verify that changes were made.
+SELECT * FROM animals;
+
+--COMMIT CHANGES
+
+COMMIT;
+
+--VERIFY IF THE CHANGES ARE PERMANENT
+
+SELECT * FROM animals;
+
```

Comment:
- [ ] Kindly add a transaction that represent this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=of%20your%20actions.-,Inside%20a%20transaction%3A,Commit%20transaction,-Take%20a%20screenshot). Don't forget to Take a screenshot of the results of your actions.